api/RTCPeerConnectionIceErrorEvent.json

```
@@ -185,6 +234,54 @@
               "version_added": "77"
             }
           },
+          "status": {
+            "experimental": true,
```

Ah, this was actually the last part of the `hostCandidate` entry. I see that the guideline actually doesn't say it, but I don't think it makes sense to call something both experimental and deprecated. It was experimental, didn't get implemented outside of Chromium, and now it's deprecated, no longer experimental. So:

```suggestion
            "experimental": false,
```

(I think we should lint again things being both experimental and deprecated.)